lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -246,9 +253,10 @@
       data-display-name="${_("Discussion Community TAs")}"
       data-info-text="
         ${_("Community TA's are members of the community whom you deem particularly "
```

Comment:
One of the goals for this PR and story was to minimize changes to the strings, since they have already been translated into multiple languages. I tried to avoid editing any text that wasn't directly affected by the changed role names.